mod.ts

```diff
@@ -68,6 +68,11 @@ export type CatppuccinFlavor = Readonly<{
    */
   name: string;
 
+  /**
+   * Emoji associated with the flavor
+   */
```

Comment:
It'd probably be nice to include the minimum emoji standard version required for all four emoji's to render properly right? (https://unicode.org/emoji/charts/emoji-versions.html)

Reply:
Hmm. I would say that it is pretty much guaranteed for the support to be there? 3 of the emojis were all part of the first emoji standard, and the newest is 🪴 which was approved as part of [Unicode 13.0](https://emojipedia.org/unicode-13.0) in 2020 and added to [Emoji 13.0](https://emojipedia.org/emoji-13.0) in 2020.

Reply:
(Using https://emojipedia.org/ to find the Unicode/emoji standard versions works better than that chart you linked FYI.)

Reply:
Yeah I think saying in the docstring that Unicode 13.0 and above is required for it to render is worthwhile information?

Reply:
Will do.